### DISCUSSION TITLE
Welcome to Larasonic Discussions!

### DISCUSSION THREAD

**Discussion by pushpak1300** [shipfastlabs/larasonic-vue#2] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we
- build together 💪.
- If you built something with Larasonic don't forget to tell in [#show-and-tell ](https://github.com/pushpak1300/Larasonic/discussions/categories/show-and-tell)channel

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PavanPersuut** [comment]
Hi Pushpak,

Looking to build a HR website using Larasonic, can I please get a demo Link on how to customize the Larasonic framework to achieve the same. Also any tutorials on the components used and their customization will be really helpful.

**Reply by pushpak1300** [reply]
https://docs.larasonic.com/introduction
https://www.shadcn-vue.com/
It's not framework just a starter kit built on top of laravel.

**Reply by PavanPersuut** [reply]
Thank you.

**Reply by PavanPersuut** [reply]
Any good video links you suggest for developing a website using Laravel+shadcn vue+inertia which are basics to start using your Larasonic starter kit for development of the website
